tools/engine_tool/README.md

```diff
@@ -60,3 +63,9 @@ implementation, then write a fake implementation.
 * *Begin with the end in mind* - Start working from what the interface provided
 by this tool *should* be, then modify underlying scripts and tools to provide
 APIs to support that.
+
+Run tests using `//flutter/testing/run_tests.py`:
+
+```shell
+testing/run_tests.py --type dart-host --dart-host-filter flutter/tools/engine_tool
```

Comment:
This depends on your OS. On Windows this should be `--dart-host-filter flutter\tools\engine_tool` 😞

Reply:
We could add a script or scripts under `engine_tool/dev` that runs the tests? (Eventually maybe we'll add `et build //flutter/tools/engien_tool:run_tests` or something.)

Reply:
Personally I'm fine with using `run_tests.py` in the short-term with a pointer on how to do so. I'll defer to whatever y'all think is best for now though :)

Reply:
Nothing needed about this for this PR. Just jotting down the idea.